### PR TITLE
Docs issue 2192

### PR DESCRIPTION
**Pull request by alexlikho**
https://github.com/exberry-io/docs/issues/2192

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -173,7 +174,11 @@ jobs:
             echo helm package for $directory
             name=$(cat $directory/Chart.yaml | grep ^name: | cut -d: -f2 | cut -c 2- | tr -d '"')
             chart_version=$(cat $directory/Chart.yaml | grep ^version: | cut -d: -f2 | cut -c 2- | tr -d '"')
-            release_version=$chart_version-${{ env.VERSION }}
+            if [[ "${{ env.VERSION }}" == *develop* || "${{ env.VERSION }}" == *master* || "${{ env.VERSION }}" != [0-9]* ]]; then
+              release_version=$chart_version-${{ env.VERSION }}:${{ env.COMMIT_HASH }}
+            else
+              release_version=$chart_version-${{ env.VERSION }}
+            fi
```
Comment (segabriel) [inline .github/workflows/ci.yml:181]: Why do we need `|| "${{ env.VERSION }}" != [0-9]*`?

Comment (segabriel) [reply .github/workflows/ci.yml:181]: ~~You can use `${{ github.event.release.prerelease }} ` instead~~

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -214,7 +219,7 @@ jobs:
               echo "${DIRECTORY}" >> ${GITHUB_ENV}
               echo "EOF" >> ${GITHUB_ENV}
               if [[ "${{ env.VERSION }}" == *develop* || "${{ env.VERSION }}" == *master* || "${{ env.VERSION }}" != [0-9]* ]]; then
-                 export VERSION=1.0-$(git rev-parse --abbrev-ref HEAD)-$(git describe --tags --abbrev=1 2>/dev/null)-${GITHUB_RUN_ATTEMPT}
+                 export VERSION=1.0-$(git rev-parse --abbrev-ref HEAD)-$(git describe --tags --abbrev=0 2>/dev/null):${{ env.COMMIT_HASH }}-${GITHUB_RUN_ATTEMPT}
```
Comment (segabriel) [inline .github/workflows/ci.yml:222]: Why `--abbrev=1` was changed?

Comment (alexlikho) [reply .github/workflows/ci.yml:222]: As I see it, `--abbrev=1` give us some details about last commit and its commit hash prefix. But currently I have added full commit hash value => `--abbrev=0` mute previous data about commit hash prefix

Comment (segabriel) [reply .github/workflows/ci.yml:222]: And are you sure we can use `:` as a separator here? Maybe simple dash `-` is better, isn't it?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (segabriel) [reply .github/workflows/ci.yml:181]: 1) And are you sure we can use : as a separator here? Maybe simple dash - is better, isn't it?
2) Could you use explicitly `master` and `develop`? No need to pass `master2`, `2master`, `my-best-master`.

**Pull request closed without merging**